pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -634,6 +634,10 @@ func (cr *PerconaServerMongoDB) MongosNamespacedName() types.NamespacedName {
 }
 
 func (cr *PerconaServerMongoDB) CanBackup() error {
+	if cr.Spec.Unmanaged {
```

Comment:
do we need this check there?
We already return error on CheckNSetDefaults

Reply:
Yes, we do. We're returning the error in `CheckNSetDefaults` if `spec.backup.enabled` is true. User can always create a backup object even if it's false.